DependencyInjection/CmfSeoExtension.php

```diff
@@ -81,8 +81,17 @@ public function load(array $configs, ContainerBuilder $container)
             $this->loadSonataAdmin($config['sonata_admin_extension'], $loader, $container, $sonataBundles);
         }
 
-        if ($this->isConfigEnabled($container, $config['alternate_locale'])) {
-            $this->loadAlternateLocaleProvider($config['alternate_locale'], $container);
+        if ($config['content_listener']) {
```

Comment:
@dbu @WouterJ @ElectricMaxxx 

This is the current state of the logic.

The default value of content_listener is cmf_seo.event_listener.seo_content_default
If this is the value of the config key then we load the content listener service definition (which is a tagged service, we dont want to do load file unless this is the listener in use or we would have multiple kernel request listeners)
If the user provides another service id, they are responsible for tagging it at as a kernel request listener. We could do this automatically but I believe it would have to be in compiler pass because the service may be from another bundle. I think we may be best off just allowing disabling of the content listener and provide doc instructions that you can just provide your own. The other thing about this PR is that the loadAlternateLocaleProvider functionality is only applied to the default seo content listener so the user would have to handle this in their service if they disable this listener.

At this point I am leaning towards just changing 'content_listener' to 'enable_content_listener' as a boolean and providing instructions for users to provide their own listeners. It reduces code complexity here.